### PR TITLE
Heading elements

### PR DESCRIPTION
**Pull request by petele**
Adds `<h1>` through `<h6>` elements

### CODE REVIEW

**Comment by petele** [comment]
@ddbeck & @foolip  - this is my first attempt at headings, hopefully it's not too far off!

File: features/headings.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
-name: <h1>
-description: The `<h1>` element TODO.
+name: Section headings elements
+description: The `<h1>` through `<h6>` elements represent six levels of section headings.
 spec:
```
Comment (foolip) [inline features/headings.yml:3]: There's just a single section in the spec here, with many IDs. Let's use just https://html.spec.whatwg.org/multipage/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements, which is what the ToC links to.

File: features/headings.yml [hunk 2]
```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
-name: <h1>
-description: The `<h1>` element TODO.
+name: Section headings elements
```
Comment (foolip) [inline features/headings.yml:1]: I'm unsure about naming for collections of elements. For something like `<picture>` and `<table>` I think we should name the feature after those elements, even though other elements are involved. Here, just`<h1>` doesn't represent all elements. But how about "&lt;h1>-&lt;h6>"? @ddbeck WDYT?

**Review by foolip — COMMENTED** [review]
The status here is wrong, I'm sure the elements were supported in Safari 1. But the data uses ranges in BCD, so this should get turned into ranges here too after some work that @ddbeck is doing. So let's leave it alone.

**Comment by petele** [comment]
Addressed feedback.

File: features/headings.yml (hunk 1 above)
Comment (petele) [reply features/headings.yml:3]: done

File: features/headings.yml (hunk 2 above)
Comment (petele) [reply features/headings.yml]: done

File: features/headings.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name: <h1> through <h6>
+description: The `<h1>` through `<h6>` HTML elements represent six levels of section headings, in order of decreasing importance.
+spec:
+  - https://html.spec.whatwg.org/multipage/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements
+compat_features:
```
Comment (captainbrosset) [inline features/headings.yml:5]: Let's add group:html like we've done for the other html element features.

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: features/headings.yml (hunk 3 above)
Comment (petele) [reply features/headings.yml:6]: done

File: features/headings.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+name: <h1> through <h6>
+description: The `<h1>` through `<h6>` HTML elements represent six levels of section headings, in order of decreasing importance.
+spec: https://html.spec.whatwg.org/multipage/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements
+group: html
```
Comment (captainbrosset) [inline features/headings.yml:4]:
```suggestion
group: html-elements
```

A new group just landed. So let's use it here.

**Review by captainbrosset — APPROVED** [review]

**Review by captainbrosset — APPROVED** [review]

**Comment by ddbeck** [comment]
Sorry about that. The way to fix this is to merge `main` into your branch and run `npm run dist` and commit the result. The conflicts to `.dist` files will get wiped out by `npm run dist`.

**Comment by petele** [comment]
> Sorry about that. The way to fix this is to merge `main` into your branch and run `npm run dist` and commit the result. The conflicts to `.dist` files will get wiped out by `npm run dist`.

I think I did that, but not sure I did the right thing, it looks pretty messy now, and it still has the conflict. Here's what I ran to get here.

```
git checkout main
git fetch upstream
git merge upstream/main
git checkout elem-headings
git rebase main
git ci
npm run dist
git rebase
git push
```

**Comment by petele** [comment]
Replaced by #1463 after messing this one up.

**Pull request closed without merging**